test-cases/upgrades/customer-profile/rolling-upgrade-custom-d1.yaml

```diff
@@ -0,0 +1,50 @@
+test_duration: 360
+
+# workloads
+# Customer profiles directory path is: scylla-qa-internal/customer-profile/cust_d
+
+# To be used in keyspace_entire_test:
+cs_duration: '300m'  # TODO: measure new upgrade test duration and adjust this one.
```

Comment:
is this need, if each one has it's own duration now ?

Reply:
removed